HW1/P5/P5_bfs.py

```diff
@@ -0,0 +1,61 @@
+from pyspark import AccumulatorParam
+
+class SetAccumulator(AccumulatorParam):
```

Comment:
This seems like a good approach. It's surprising to that this wasn't enough to speed up the code to find all paths and that you had to reduce to an arbitrary path to make this finish in a reasonable time. This is a good optimization though.